From 4efb353e470675da8a65cc24c8918e68c05ab9d4 Mon Sep 17 00:00:00 2001
From: Willy Tarreau <w@1wt.eu>
Date: Wed, 29 Jan 2014 12:13:39 +0100
Subject: [PATCH 82/82] BUG/MINOR: cli: fix missing break in command line
 parser

Yesterday's commit 12833bb ("MINOR: cli: add the new "show pools" command")
missed a "break" statement causing trouble to the "show map" command. Spotted
by Thierry Fournier.
---
 src/dumpstats.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/dumpstats.c b/src/dumpstats.c
index 3adfdf3..2599785 100644
--- a/src/dumpstats.c
+++ b/src/dumpstats.c
@@ -2180,6 +2180,7 @@ static void cli_io_handler(struct stream_interface *si)
 			case STAT_CLI_O_MLOOK:
 				if (stats_map_lookup(si))
 					appctx->st0 = STAT_CLI_PROMPT;
+				break;
 			case STAT_CLI_O_POOLS:
 				if (stats_dump_pools_to_buffer(si))
 					appctx->st0 = STAT_CLI_PROMPT;
-- 
1.8.3.2

